ivy/functional/frontends/numpy/mathematical_functions/rounding.py

```diff
@@ -0,0 +1,26 @@
+import ivy
+
+
```

Comment:
please add the decorator `@from_zero_dim_arrays_to_float`

Reply:
Done